### PR TITLE
Add dc coverage

### PR DESCRIPTION
**Pull request by brinxmat**

### CODE REVIEW

File: import-brage/src/main/java/no/unit/nva/importbrage/XmlImport.java [hunk 1]
```diff
@@ -24,11 +25,20 @@ private XmlImport() {
      * @return A list of contributor objects.
```
Comment (janbj) [inline import-brage/src/main/java/no/unit/nva/importbrage/XmlImport.java:25]: Update javadoc

**Review by janbj — APPROVED** [review]

File: import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageCoverage.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    public CoverageType getType() {
+        return type;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public enum CoverageType {
```
Comment (axthosarouris) [inline import-brage/src/main/java/no/unit/nva/importbrage/metamodel/BrageCoverage.java:44]: Separate file?

File: import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java [hunk 3]
```diff
@@ -15,8 +16,14 @@
 
     @Test
     void xmlImportLoadsData() throws IOException {
-        var contributors = XmlImport.map(new File(getClass().getResource(OVERCOMPLETE_EXAMPLE).getFile()));
-        contributors.forEach(this::testContributor);
+        var publication = XmlImport.map(new File(getClass().getResource(OVERCOMPLETE_EXAMPLE).getFile()));
```
Comment (axthosarouris) [inline import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java:19]: Break this test into multiple tests. The tests could fail for more than one reasons. Also values are not verified to much the expected values. I would suggest to create a smaller file where you test that the values are read correctly (and they are not for example empty strings).

**Review by axthosarouris — CHANGES_REQUESTED** [review]
See comments above

**Pull request merged**